docs/en/docs/templates.md

```diff
@@ -28,19 +28,31 @@ the `Jinja2Template`.
 ```python
 {!> ../../../docs_src/templates/template.py !}
 ```
+
 ### Templates response parameters
 
 The get_template_response function expects the following arguments:
 
-- `request`: (required): The HTTP request object.
-- `name`: (required): The name of the template to render.
+- `request` (required): The HTTP request object.
+- `name` (required): The name of the template to render.
+- `context` (optional): A dictionary allowing you to include dynamic data in the template rendering process.
+
+
+Any additional arguments or keyword arguments provided will be passed directly to `TemplateResponse`.
+
+### Optional Arguments
+
+- `status_code` (int, optional): The status code of the response. Defaults to 200.
+- `background` (BackgroundTask, optional): An instance of BackgroundTask. Defaults to None.
+- `headers` (dict[str, Any], optional): A dictionary of response headers. Defaults to None.
+- `media_type` (str, optional): The media type of the response. Defaults to "text/html".
+
+       
 
-Any additional arguments or keyword arguments provided will be passed directly to the template as context.
-This allows you to include dynamic data in the template rendering process.
 You can pass these arguments either as keyword arguments or positional arguments, depending on your preference.
 
 !!! warning
-    It's imperative to include the incoming request instance as part of the template context.
+It's imperative to include the incoming request instance as part of the template context.
```

Comment:
give it a tab (4 spaces) here please, this is markdown and will show the text inside the warning.

Reply:
fixed